share/p2p/shrexnd/pb/share.proto

```diff
@@ -4,7 +4,7 @@ package share.p2p.shrex.nd;
 
 message GetSharesByNamespaceRequest{
   bytes root_hash = 1;
-  bytes namespace_id = 2;
+  bytes namespace = 2;
```

Comment:
Not that naming changes in PB are not protocol breaking, as long as type and sequence are the same